tests/cft/test_s3bench_workload.py

```diff
@@ -40,24 +40,25 @@ def setup_class(cls):
         cls.cft_test_cfg = configmanager.get_config_wrapper(fpath=test_config)
         cls.setup_type = CMN_CFG["setup_type"]
 
-    @pytest.mark.longevity
-    @pytest.mark.tags("TEST-19658")
-    def test_19658(self):
-        """Longevity Test with distributed workload"""
-        test_cfg = self.cft_test_cfg["test_12345"]
+    def execute_workload_distribution(self, test, log_file_prefix):
```

Comment:
do we want to shift to lib maybe? so in future if anyone else wants similar workload?

Reply:
Will keep this as is for now.